auction-app-frontend/src/components/Searchbar/Searchbar.jsx

```diff
@@ -0,0 +1,45 @@
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+import { search } from "src/assets/icons";
+
+import "./style.scss";
+
+const Searchbar = () => {
+  const [query, setQuery] = useState("");
```

Comment:
It's too generic, this is product name, so why not `[productName, setProductName]`